static/js/libs/events.js

```diff
@@ -0,0 +1,64 @@
+const Events = {
```

Comment:
Not sure why do we need that... I see that it helps with assigning many handlers for different selectors at once, but I'm just not sure that creating our own event abstraction is that helpful.